### PR TITLE
Detect properties declared in interface

### PR DESCRIPTION
**Pull request by paulbalandan**
Fixes https://github.com/phpstan/phpstan/issues/8915

### CODE REVIEW

**Comment by ondrejmirtes** [comment]
Lint fails - path to this file needs to be added as `--exclude` in the Makefile.

File: tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+
+	protected function getRule(): Rule
+	{
+		return new PropertiesInInterfaceRule();
+	}
+
+	public function testRule(): void
+	{
+		$this->analyse([__DIR__ . '/data/bug-8915.php'], [
```
Comment (mvorisek) [inline tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php]: https://3v4l.org/nstX1 in another test file would be good to test too

File: tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	protected function getRule(): Rule
+	{
+		return new PropertiesInInterfaceRule();
+	}
+
+	public function testRule(): void
+	{
+		$this->analyse([
+			__DIR__ . '/data/properties-in-interface.php',
+			__DIR__ . '/data/bug-8915.php',
```
Comment (ondrejmirtes) [inline tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php]: Same point here - separate test method pleaase :) (And FYI personally I wouldn't add this file here at all, the rule is straightforward and the test does not really test anything different :))

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php (hunk 2 above)
Comment (paulbalandan) [reply tests/PHPStan/Rules/Properties/PropertiesInInterfaceRuleTest.php]: Thanks for the pointers! I retained the first test.

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**